TwicketSegmentedControl/TwicketSegmentedControl.swift

```diff
@@ -9,7 +9,7 @@
 import UIKit
 
 public protocol TwicketSegmentedControlDelegate: class {
-    func didSelect(_ segmentIndex: Int)
+    func twicketSegmentedControl(_ segmentedControl: TwicketSegmentedControl, didSelect segmentIndex: Int)
```

Comment:
I don't think we should write stuff like UIKit just for consistency when we know they are not correct